scripts/cnode-helper-scripts/prereqs.sh

```diff
@@ -529,6 +530,7 @@ updateWithCustomConfig "gLiveView.sh"
 updateWithCustomConfig "topologyUpdater.sh"
 updateWithCustomConfig "logMonitor.sh"
 updateWithCustomConfig "cncli.sh"
+updateWithCustomConfig "blockPerf.sh"
```

Comment:
This could mess up the contents of blockPerf.sh (as there isnt a text " Do NOT modify" in the file), the grep/awk commands will probably not make intended changes

Reply:
there wasn't the first "User Variables - Change as desired " (now it is) 
but the "Do NOT modify" was there already. so should be fine now?